third_party/glad/BUILD.gn

```diff
@@ -0,0 +1,76 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+_glad_gen_dir = "$root_out_dir/gen/flutter/third_party/glad"
+
+config("glad_public") {
+  include_dirs = [ "$_glad_gen_dir/include" ]
+}
+
+source_set("glad") {
+  public_configs = [ ":glad_public" ]
+
+  public = [
+    "$_glad_gen_dir/include/KHR/khrplatform.h",
+    "$_glad_gen_dir/include/glad/glad.h",
+  ]
+
+  sources = [
+    "$_glad_gen_dir/src/glad.c",
+  ]
+
+  deps = [
+    ":gen_glad_sources",
+  ]
+}
+
+action("gen_glad_sources") {
```

Comment:
This rule should have file-private visibility.

Reply:
Sorry, I don't quite understand how to modify it.

Reply:
See the first example [here](https://gn.googlesource.com/gn/+/master/docs/reference.md#var_visibility)